flixel/addons/editors/pex/FlxPexParser.hx

```diff
@@ -54,7 +55,7 @@ class FlxPexParser
 		var angle = minMax("angle", config);
 		
 		var startSize = minMax("startParticleSize", config);
-		var finishSize = minMax("finishParticleSize", "FinishParticleSizeVariance", config);
+		var finishSize = minMax("finishParticleSize", "finishParticleSizeVariance", config);
```

Comment:
Are you sure about this? In the `FlxPexParser` demo, the `.pex` file actually uses `FinishParticleSizeVariance`, and changing this breaks it:

```
particleEmitterConfig is missing element finishParticleSizeVariance
	at haxe.xml._Fast::NodeAccess/resolve()
	at flixel.addons.editors.pex::FlxPexParser$/minMax()
	at flixel.addons.editors.pex::FlxPexParser$/parse()
```